src/js/actions/addScale.js

```diff
@@ -9,7 +9,7 @@ function addScale(data) {
   }, data);
 
   var action = {
-    id: props._id,
+    id: 'scale_' + props._id,
```

Comment:
The `get` and `set` utilities I wrote coerce all keys to strings, and were intended to resolve that issue